packages/portal/frontend/src/app/views/AdminConfigTab.ts

```diff
@@ -417,7 +417,7 @@ export class AdminConfigTab extends AdminPage {
 
     private async updateClasslistPressed(): Promise<void> {
         Log.trace('AdminConfigTab::updateClasslistPressed(..) - start');
-        const url = this.remote + '/portal/admin/classlist';
+        const url = this.remote + '/portal/classlist';
```

Comment:
Shouldn't this route be an admin route? Looking at all of the other routes in `AdminRoutes.ts` it seems like this should be included there and be called `/portal/admin/classlist`?

Reply:
The intent is to make this something that anything can poke, including an internal techstaff process that pokes this endpoint when your classlist has changed. We protect from denial of service by restricting access by IP.